code/modules/paperwork/paper.dm

```diff
@@ -400,6 +400,24 @@
 	SHOULD_CALL_PARENT(FALSE)
 	return
 
+/obj/item/paper/crumpled/engineeringnotice
```

Comment:
Im pretty sure this was meant for the Tesla PR, remove please

Reply:
Guh